util/pkg/vfs/gsfs.go

```diff
@@ -33,6 +34,7 @@ import (
 	storage "google.golang.org/api/storage/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/klog/v2"
+	"k8s.io/kops/upup/pkg/fi/cloudup/terraformWriter"
```

Comment:
Was the mixed-case deliberate here?  It feels like the sort of thing which might cause problems, so I would probably have used simply `terraformwriter`

Reply:
The package itself is mixed case. we could certainly modify that in a followup PR.

https://github.com/kubernetes/kops/tree/master/upup/pkg/fi/cloudup/terraformWriter